lib/l10n/intl_en.arb

```diff
@@ -1177,6 +1177,30 @@
   "@demoCupertinoButtonsDescription": {
     "description": "Description for the cupertino buttons component demo."
   },
+  "demoCupertinoContextMenuTitle": "Context Menu",
+  "@demoCupertinoContextMenuTitle": {
+    "description": "Title for the cupertino context menu component demo."
+  },
+  "demoCupertinoContextMenuSubtitle": "iOS-style context menu",
+  "@demoCupertinoContextMenuSubtitle": {
+    "description": "Subtitle for the cupertino context menu component demo."
+  },
+  "demoCupertinoContextMenuDescription": "A full-screen modal route that opens when the child is long-pressed. When open, the CupertinoContextMenu shows the child, or the widget returned by previewBuilder if given, in a large full-screen Overlay with a list of buttons specified by actions.",
```

Comment:
Simplify this description a bit . Something like:

"An iOS-style full screen contextual menu that appears when an element is long-pressed."

Reply:
I've replaced it with this one .